Source/WebCore/css/values/primitives/CSSUnevaluatedCalc.h

```diff
@@ -189,6 +189,12 @@ template<Calc T> struct CSSValueChildrenVisitor<T> {
 } // namespace CSS
 } // namespace WebCore
 
-template<WebCore::CSS::Calc T> struct WTF::IsSmartPtr<T> {
+namespace WTF {
+template<typename T>
+struct IsSmartPtr;
+
+template<WebCore::CSS::Calc T> struct IsSmartPtr<T> {
```

Comment:
I think a better fix for this would be to either:

a) Add the include for <wtf/GetPtr.h> to these files.
b) Add IsSmartPtr forward declaration to wtf/Forward.h and make sure wtf/Forward.h is included in these files.

Reply:
Thanks for the suggestions, you're right.

Reply:
Turns out the culprit is not having the namespace scope in the instantiation of the template, but I added the forward-declaration just for extra clarity as you suggested.

Reply:
Are both really needed? I would expect just the forward declaration would be enough. Either way, thanks!

Reply:
Yeah, somehow the scoped namespace is needed too. Thanks to you!